jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-backend/src/test/java/org/jbpm/console/ng/cm/backend/server/RemoteCaseManagementServiceImplTest.java

```diff
@@ -174,4 +177,62 @@ public void testGetCaseInstance() throws Exception {
         assertCaseInstance(ci, cis);
     }
 
+    @Test
+    public void testGetCaseComments() throws Exception {
+        final CaseComment caseComment = new CaseComment();
+        caseComment.setId("CommentId");
+        caseComment.setAuthor("Author");
+        caseComment.setText("Text");
+        caseComment.setAddedAt(new Date());
+
+        when(caseServicesClient.getComments(anyString(), anyString(), anyInt(), anyInt())).thenReturn(
+                Collections.singletonList(caseComment));
+
+        final List<CaseCommentSummary> comments = service.getComments("id", "containerId", "caseId", 0, 0);
+        assertNotNull(comments);
+        assertEquals(1, comments.size());
+        final CaseCommentSummary caseCommentSummary = comments.get(0);
+        assertEquals(caseComment.getId(), caseCommentSummary.getId());
+        assertEquals(caseComment.getAuthor(), caseCommentSummary.getAuthor());
+        assertEquals(caseComment.getText(), caseCommentSummary.getText());
+        assertEquals(caseComment.getAddedAt(), caseCommentSummary.getAddedAt());
+    }
+
+    @Test
+    public void testAddComment() throws Exception {
+        final String caseId = "org.jbpm";
```

Comment:
Why not extract these (caseId, container, author ect.) as class constants the way you do in `CaseCommentsPresenterTest`